server/src/main/java/org/opensearch/http/AbstractHttpServerTransport.java

```diff
@@ -85,6 +91,8 @@
 public abstract class AbstractHttpServerTransport extends AbstractLifecycleComponent implements HttpServerTransport {
     private static final Logger logger = LogManager.getLogger(AbstractHttpServerTransport.class);
     private static final ActionListener<Void> NO_OP = ActionListener.wrap(() -> {});
+    private static final List<String> HEADERS_TO_BE_POPULATED = Arrays.asList(AttributeNames.TRACE);
```

Comment:
What these headers are for?

Reply:
These are list of headers that we want to add as Attributes like "trace=true" for on demand sampling.